### PR TITLE
workload: more correct stats names

### PR DESCRIPTION
**Pull request by AndrewMeadows**

### CODE REVIEW

**Comment by AndrewMeadows** [comment]
@samcake, this is for you: more correct stats so true performance bottlenecks are easier to identify understand.

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/8756/app_12880_PR_6e6b90fec47e6f65bd642fb6764c0f85877ace42.apk

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/8756/HighFidelity-Beta-PR12880-6e6b90fec47e6f65bd642fb6764c0f85877ace42.dmg

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/8759/app_12880_PR_d553f85e83928cd5b743e1a619dec529b3cb4076.apk

File: interface/src/Application.cpp [hunk 1]
```diff
@@ -5272,12 +5272,12 @@ void Application::update(float deltaTime) {
...
-        PROFILE_RANGE(simulation_physics, "Physics");
-        PerformanceTimer perfTimer("physics");
+        PROFILE_RANGE(simulation_physics, "Simulation");
+        PerformanceTimer perfTimer("simulation");
         if (_physicsEnabled) {
             {
-                PROFILE_RANGE(simulation_physics, "PreStep");
-                PerformanceTimer perfTimer("preStep)");
+                PROFILE_RANGE(simulation_physics, "PrePhysics");
+                PerformanceTimer perfTimer("prePhys)");
```
Comment (samcake) [inline interface/src/Application.cpp]: missing ics ?

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/8759/HighFidelity-Beta-PR12880-d553f85e83928cd5b743e1a619dec529b3cb4076.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/8759/HighFidelity-Beta-Interface-PR12880-d553f85e83928cd5b743e1a619dec529b3cb4076.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/8759/HighFidelity-Beta-PR12880-d553f85e83928cd5b743e1a619dec529b3cb4076.exe

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/8762/app_12880_PR_6633304773049d84dea6826a4d76a9939c94e898.apk

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/8762/HighFidelity-Beta-PR12880-6633304773049d84dea6826a4d76a9939c94e898.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/8762/HighFidelity-Beta-Interface-PR12880-6633304773049d84dea6826a4d76a9939c94e898.exe

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/8762/HighFidelity-Beta-PR12880-6633304773049d84dea6826a4d76a9939c94e898.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/8762/HighFidelity-Beta-Interface-PR12880-6633304773049d84dea6826a4d76a9939c94e898.dmg

**Pull request merged**